src/views/sponsorship.html

```diff
@@ -81,7 +81,7 @@ <h1 id="sponsorship-hack-cambridge-wordmark">Hack<br>Cambridge<br><span class="p
                     </div>
                     <div class="half unit row shown"></div>
                     <div id="sponsorship-university-of-cambridge-endorsement" class="half unit row centred">
-                        <img id="sponsorship-university-of-cambridge-logo-object" src="{{ asset('images/hc-2018/university-of-cambridge-logo.svg') }}"></object>
+                        <img id="sponsorship-university-of-cambridge-logo-object" src="{{ asset('images/hc-2018/university-of-cambridge-logo.svg') }}"></img>
```

Comment:
Use `<img />` since img is a void element and in HTML5 [void elements must not have end tags](http://w3c.github.io/html-reference/syntax.html#void-element)